Signal/src/call/PeerConnectionClient.swift

```diff
@@ -359,7 +359,11 @@ class PeerConnectionClient: NSObject, RTCPeerConnectionDelegate, RTCDataChannelD
                 }
                 Logger.verbose("\(self.TAG) setting local session description: \(sessionDescription)")
                 self.peerConnection.setLocalDescription(sessionDescription.rtcSessionDescription,
-                                                        completionHandler: { _ in
+                                                        completionHandler: { error in
+                                                            guard error == nil else {
+                                                                reject(error!)
```

Comment:
I've never personally seen this fail, but it seems wrong that we would ignore these errors.